test/tx-builder/dydx-governance.test.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
+import hre from 'hardhat';
+
+import { Network, tDistinctGovernanceAddresses, TxBuilder } from '../../src';
+import config from '../../src/config';
+
+describe('DydxGovernance', () => {
+  const blockAtWhichVotersAreExpected: number = 13678600;
+  const governanceAddress: string = '0x7E9B1672616FF6D6629Ef2879419aaE79A9018D2';
```

Comment:
You shouldn't need to hardcode this address if you use the [`TestContext`](https://github.com/dydxfoundation/governance-contracts/blob/b06718fe1639d2be4acc02214139dbe6e104a32b/test/helpers/describe-contract.ts#L15) object that's included in the `describeContractForNetwork` wrapper.